asset/cpu_triggered_asset_utilization.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import sys
+import time
+import winreg
+
+LOGGED_USER_NAME = getpass.getuser()
+
+parser = argparse.ArgumentParser(description = 'CPU usage triggered asset utilization.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('filename', nargs ='+', action = 'store')
+parser.add_argument('-threshold', '-t', type = int, default = 15, help = 'The threshold percentage that triggers the utilization start.')
```

Comment:
In general, long args use double dash, while short ones use single dash (--threshold, -t)